src/components/projects/A32NX/A32NXProjHeader.tsx

```diff
@@ -3,9 +3,9 @@ import BackgroundImage from '../../../assets/img/discord.jpg';
 import a32nx_wireframe from '../../../assets/svg/a32nx_wireframe.svg';
```

Comment:
This doesn't need to be renamed to specifics as it's already in a folder called A32NX

Reply:
Hmm, maybe we could change it to something else. I still don't think that "Hero" makes sense, either.

Reply:
But that's what they're called

Reply:
yes yes you are correct, thank you for that; reverting back to 4645863